src/integTest/resources/loose-ear-test/ejb-ear/build.gradle

```diff
@@ -17,13 +17,17 @@ buildscript {
     }
 }
 
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
 
 dependencies {
     deploy project(':ejb-ejb')
     deploy project(path:':ejb-war', configuration:'archives')
     testCompile group: 'commons-httpclient', name: 'commons-httpclient', version:'3.1'
     testCompile group: 'junit', name: 'junit', version:'4.12'
-    libertyRuntime group: 'com.ibm.websphere.appserver.runtime', name: 'wlp-javaee7', version: '17.0.0.2'
+    libertyRuntime group: runtimeGroup, name: runtimeArtifactId, version: runtimeVersion
```

Comment:
This test used to use wlp-javaee7. Now it will use wlp-webProfile7 because of the  properties file, correct? Is that intentional?